front/src/modules/scenario/components/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -110,7 +131,7 @@ const ScenarioExplorer = ({
       globalStudyId &&
       studyDetails &&
       globalScenarioId &&
```

Comment:
No need to check if the ids are defined, can you clean this please ? :) 
```suggestion
      studyDetails &&
```